view/templates/email/html.tpl

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional //EN">
+<html>
+<head>
+	<title>{{$banner}}</title>
+	<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
+</head>
+<body>
+	<table style="border:1px solid #ccc">
+	<tbody>
+		<tr>
+			<td colspan="2" style="background:#084769; color:#FFFFFF; font-weight:bold; font-family:'lucida grande', tahoma, verdana,arial, sans-serif; padding: 4px 8px; vertical-align: middle; font-size:16px; letter-spacing: -0.03em; text-align: left;">
+				<img style="width:32px;height:32px; float:left;" src='{{$siteurl}}/images/friendica-32.png'>
```

Comment:
I think this image should only the fall back for the case when no banner/logo was set for the Friendica instance by the admin.

Reply:
The banner is a "textarea", where - per default - every kind of HTML/CSS is allowed. I think if we open the possibility (again?) to map the banner to the mails, it could break the layout because the banner is mostly optimized for the frontend, not for the emails.

I'd say we have to add a new setting "banner_email" and force image source only :)

Reply:
I added a new `system.email_banner` config for the admin and used it for the emails now :)